pkg/graph/schema.graphql

```diff
@@ -431,6 +432,8 @@ PlanGeneralCharacteristics represents a plan general characteristics object
 type PlanGeneralCharacteristics {
   id: UUID!
   modelPlanID: UUID!
+  demoCode: String
+  amsModelID: String

```

Comment:
I think we will need to add these fields to [PlanGeneralCharacteristicsChanges](https://github.com/CMSgov/mint-app/pull/590/files#diff-7b3b137cd7a3e86ef5717c83df62cd6be4aa416eb0d4d7eb1ebc7ecba3bf92c4L517) as well so that these can be modified/set be the frontend